### DISCUSSION TITLE
[SOFAR] 3phase works, 1phase doesn't?

### DISCUSSION THREAD

**Discussion by zkhcj** [davidrapan/ha-solarman#165] [Q&A]
Hi all,

So, I have 2 different inverters. One 3 phase and one 1 phase. The 3 phase works just fine, gives me all the info I need (80 sensors)
<img width="545" alt="image" src="https://github.com/user-attachments/assets/9efb406a-2ec7-4be7-ab41-2327d71645b3">

The 1phase doesn't. It gives errors all the time and it also does a bit of a weird thing with the country it's in. In the solar man app both are configured the same, I tried other configs (instead of the sofar_lsw3.yaml) but nothing changes. Anyone have an idea why it's not working? The inverter is not only giving me errors, but more important, isn't showing what it has produced during the day, nor the current status. However, in the solar man app it works just fine.
<img width="544" alt="image" src="https://github.com/user-attachments/assets/c7328374-90b3-4f36-8348-ecada728d0d0">
<img width="320" alt="image" src="https://github.com/user-attachments/assets/766d0e99-bdbf-42a1-8896-68ee1ddc467d">

Any ideas?

**Comment by davidrapan** [comment]
Hi @zkhcj,

What is the exact model of that 1P device? And can you show it's device info page which contains fw and protocol version and such from Solarman Smart App?

**Reply by zkhcj** [reply]
Hi David, is this what you’re looking for?
![IMG_0552](https://github.com/user-attachments/assets/0f609bf3-197f-4657-8436-9de5843a3886)

![IMG_0553](https://github.com/user-attachments/assets/60b4b3f2-a8eb-47e4-a4f9-b04a686831b1)

**Reply by davidrapan** [reply]
Yes that's it! Hmm maybe it's a long shot but... couldn't that be similar "issue" as in #142? The thread is about ?KTL-X-G3 but the guy also did describe strange behavior with the country sensor and he was able to resolve his issue by asking support to upgrade FW of his device...
